app/src/features/onboarding/components/teams/index.tsx

```diff
@@ -35,6 +35,8 @@ export const WorkspaceOnboardingView: React.FC<WorkspaceOnboardingViewProps> = (
     []
   );
 
+  const getUserTeams = useMemo(() => httpsCallable(getFunctions(), "teams-getUserTeams"), []);
+
```

Comment:
Instead of calling the cloud function, can we use the `getAvailableTeams` instead?